app/forms/state_file/nj_ineligible_property_tax_form.rb

```diff
@@ -0,0 +1,5 @@
+module StateFile
+  class NjIneligiblePropertyTaxForm < QuestionsForm
+    def save; end
```

Comment:
are these empty save methods necessary (here and in nj_unsupported_property_tax_form)?